packages/prosess-aarskvantum-oms/src/components/Uttaksplan.tsx

```diff
@@ -50,28 +52,21 @@ const Uttaksplan: FunctionComponent<UttaksplanProps> = ({
         </>
       }
     >
-      {aksjonspunkterForSteg.length > 0 && (
-        <AksjonspunktForm
-          aktiviteter={aktiviteter}
-          behandlingId={behandlingId}
-          behandlingVersjon={behandlingVersjon}
-          submitCallback={submitCallback}
-          aksjonspunkterForSteg={aksjonspunkterForSteg}
-          isAksjonspunktOpen={isAksjonspunktOpen}
-        />
-      )}
-      {aktiviteter.length ? (
-        aktiviteter.map(({ arbeidsforhold, uttaksperioder }) => (
-          <AktivitetTabell
-            arbeidsforhold={arbeidsforhold}
-            uttaksperioder={uttaksperioder}
-            aktivitetsstatuser={aktivitetsstatuser}
-            key={joinNonNullStrings(Object.values(arbeidsforhold))}
-          />
-        ))
-      ) : (
-        <FormattedMessage id="Uttaksplan.IngenUttaksplaner" />
-      )}
+      <Tabs
+        tabs={[
+          { label: <FormattedMessage id="Uttaksplan.DenneBehandling" /> },
+          { label: <FormattedMessage id="Uttaksplan.HittilIÅr" /> },
+        ]}
+        onChange={(e, valgtIndex) => setValgtTabIndex(valgtIndex)}
+      />
+      <VerticalSpacer sixteenPx />
+      {valgtTabIndex === 0 &&
```

Comment:
Denne var litt tung å lese for meg som ikke kjenner koden 😕  

Kan den forenkles til noe som dette?
```
const harAktiviteterÅRendre = Array.isArray(aktiviteterBehandling) && aktiviteterBehandling.length > 0;
return (
   // something
   {harAktiviteterÅRendre ? <Aktivitetstabell /> : <FormattedMsg />}
   // something
)
```